src/docs/build-and-earn/testnet-guides/prerequisites.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Prerequisites and Operating Systems
+---
+
+# ﻿Prerequisites and Operating Systems
+
+You can run your node in different operating systems such as Debian 10, Ubuntu, WSL, and macOS Catalina version 10.15.4 or above. If you are installing docker for the first time, please check the next section, “Notes for Installing Docker for the First Time”.
+
+The first step is to ensure that you have set up all requirements and prerequisites as the following:
+
+- Install [Docker](https://docs.docker.com/engine/install/) 20.10.6 or above
+- Install [Docker Compose](https://docs.docker.com/compose/install/) 1.29.1 or above
+- Install [Git](https://github.com/git-guides/install-git)
+- Install a code editor such as Nano, Atom, etc.
+
+Missing any of the prerequisites above can lead to errors and being unable to run your node. It is recommended to have the latest version of Docker and Docker compose installed. If you already installed them, you can check the version by running the following commands in your terminal:
+```
+docker -v
+docker-compose -v
+```
+
+## Setup the environment before running a node
+
+After installing all prerequisites, you can start to set up the environment:
+1. Create a directory to use as your workspace:
+```
+mkdir ~/cudos
+cd ~/cudos
+```
+2. Make sure that you are in the correct directory (cudos directory in this example)
+3. Clone the correct branches from the [CudosNode](https://github.com/CudoVentures/cudos-node), [CudosBuilders](https://github.com/CudoVentures/cudos-builders), and [CudosGravityBridge](https://github.com/CudoVentures/cosmos-gravity-bridge) repositories:
+```
+git clone --depth 1 -b testnet https://github.com/CudoVentures/cudos-node.git CudosNode
+git clone --depth 1 -b master  https://github.com/CudoVentures/cudos-builders.git CudosBuilders
+git clone --depth 1 -b master https://github.com/CudoVentures/cosmos-gravity-bridge.git CudosGravityBridge
+```
+4. Change to the root of each local repository and check out the following branches:
```

Comment:
this step no. 4 is probably redundant and can be avoided, ie, prev step no. 3 could be:
```
git clone --depth 1 -b gravity/sdk-0.43 https://github.com/CudoVentures/cudos-node.git CudosNode
git clone --depth 1 -b sdk-0.43  https://github.com/CudoVentures/cudos-builders.git CudosBuilders
git clone --depth 1 -b cosmos-sdk-0.43 https://github.com/CudoVentures/cosmos-gravity-bridge.git CudosGravityBridge
```